docs/topics/auth/default.txt

```diff
@@ -1691,7 +1691,25 @@ provides several built-in forms located in :mod:`django.contrib.auth.forms`:
 
 .. class:: UserCreationForm
 
-    A form for creating a new user.
+    A form for creating a new user
+
+    **Attributes:**
+
+    * ``username``:
+        The username sent to the form.
```

Comment:
I'm not sure this description is adding much value. Maybe it's enough to list the fields without explanation and then add something like "The form verifies that the user entered the same password in password1 and password2."